content/english/facts/soy-contains-estrogen.md

```diff
@@ -0,0 +1,64 @@
+
+or
+ 
+> Soy contains *phytoestrogens* which are completely different and have no effect on humans
+
+# Reality 
+
+> Soy contains phytoestrogens which may interact with the human body in a variety of ways (including hormonally).   
+> - TalkVeganToMe
+
+See below for a more detailed explanation.  
+See our article on [Soy is bad for you?]({{<ref "/facts/soy-is-bad-for-you">}}) for more on what this means.  
+
+## What are phytoestrogens and isoflavones?
+> Phytoestrogens are naturally-occurring plant compounds that are structurally and/or functionally similar to mammalian estrogens and their active metabolites.
+> - [The pros and cons of phytoestrogens - Front Neuroendocrinol - 2010 - p2](https://web.archive.org/web/20190928181945/https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3074428/pdf/nihms271669.pdf)
+
+### What Isoflavones does soy contain?
+> Soy contains the isoflavones:  
+> - Biochanin A   
+> - Clycitein   
+> - Daidzein  
+> - Formononetin  
+> - Genistein    
+> - [The pros and cons of phytoestrogens - Front Neuroendocrinol - 2010 - p2](https://web.archive.org/web/20190928181945/https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3074428/pdf/nihms271669.pdf)
+
+## How do isoflavones interact with the body?
+> Some isoflavones, most notably genistein, inhibit pathways important for cell growth and proliferation, an effect which affects multiple organ systems.  
+> - [The pros and cons of phytoestrogens - Front Neuroendocrinol - 2010 - p4](https://web.archive.org/web/20190928181945/https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3074428/pdf/nihms271669.pdf)
+
+In other words, it's possible for human cell growth to be increased by phytoestrogens.
+
+> Phytoestrogens are plant‐derived dietary compounds with structural similarity to 17‐β‐oestradiol (E2), the primary female sex hormone. This structural similarity to E2 enables phytoestrogens to cause (anti)oestrogenic effects by binding to the oestrogen receptors.  
+> [...]  
+> [P]hytoestrogens are also considered endocrine disruptors, indicating that they have the potential to cause adverse health effects such as infertility and increased risks of cancer in oestrogen-sensitive organs.  
+> - [The potential health effects of dietary phytoestrogens - British Journal of Pharmacology](https://web.archive.org/web/20190922065924/https://bpspubs.onlinelibrary.wiley.com/doi/full/10.1111/bph.13622)  
+
+Phytoestrogens have a similar shape to human estorogen, which enables them to interrupt the normal absorption of human estrogen in the endocrine (hormonal) system.
+
+> Phytoestrogens produce estrogenic, anti-estrogenic or no activity depending upon the tissue. [Oestrogen Receptor Alpha] and [Oestrogen Receptor Beta] display distinct expression patterns in male and female[s...].  
+> - [Soy, Soy Foods and Their Role in Vegetarian Diets - Nutrients - 2018 - p21](https://web.archive.org/web/20191002195620/https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5793271/pdf/nutrients-10-00043.pdf/)
```

Comment:
Minor point. Can't find the text on page 21 ;) But maybe I'm being blind?

Reply:
That's because it's actually on page 7! good catch!